src/cmd/linuxkit/pkglib/docker.go

```diff
@@ -166,3 +177,142 @@ func (dr dockerRunner) save(tgt string, refs ...string) error {
 	args := append([]string{"image", "save", "-o", tgt}, refs...)
 	return dr.command(args...)
 }
+
+func (dr dockerRunner) manifestPush(img string, trust bool) error {
```

Comment:
I may be missing this, but manifestPush() does not need to be a method (or whatever it is called) of dockerRunner, right?

Actually, can't you ditch the `trust` argument and use `dr.dct` in the function directly?

Reply:
> manifestPush() does not need to be a method (or whatever it is called) of dockerRunner, right?

You are correct. I will remove it from there.

> Actually, can't you ditch the trust argument and use dr.dct in the function directly?

I like your earlier approach of making it an independent func.

Reply:
Done